src/gui/dialogs/game_version.cpp

```diff
@@ -124,26 +126,26 @@ tgame_version::tgame_version()
 	generate_plain_text_report();
 }
 
-void tgame_version::pre_show(twindow& window)
+void game_version::pre_show(window& window)
 {
 	string_map i18n_syms;
 
 	//
 	// General information.
 	//
 
-	tcontrol& version_label = find_widget<tcontrol>(&window, "version", false);
+	control& version_label = find_widget<control>(&window, "version", false);
```

Comment:
I would expect control to be an MVC-style controller, but I don't think that makes sense at all in this context. Hence, a better name for the type might be useful.